exchanges/graphcache/src/store/store.ts

```diff
@@ -78,6 +78,13 @@ export class Store implements Cache {
       if (this.keys) {
         SchemaPredicates.expectValidKeyingConfig(this.schema, this.keys);
       }
+
```

Comment:
This is looking awesome already, just a minor NIT I would wrap this whole validation part in `if (process.env.NODE_ENV !== 'production)` since these are intended as developer-time warnings rather than production ones

Reply:
Agreed - done!